tree/ntuple/v7/doc/specifications.md

```diff
@@ -1052,6 +1052,17 @@ The limits refer to a single RNTuple and do not consider combinations/joins such
 | Maximum string length (meta-data)              | 4GB                          | String encoding                                        |
 | Maximum RBlob size                             | 128 PiB                      | 1GiB / 8B * 1GiB (with maxKeySize=1GiB, offsetSize=8B) |
 
+## Defaults
+
+This section summarizes default settings of
+[`RNTupleWriteOptions`](https://github.com/root-project/root/blob/96989396d03d17b877ecc3d7eb7293109d24e5cf/tree/ntuple/v7/inc/ROOT/RNTupleWriteOptions.hxx#L61-L75).
```

Comment:
@jblomer @couet Is this the best way to refer to a header file?

Reply:
I suppose the downside of it is that we need to update it when we update the defaults. But at least this link won't break if that header file gets moved/renamed

Reply:
> I think we don't need to link the header file here. It's enough to mention the class name.

Exactly.

Reply:
How to mention it so that doxygen will point to the class? What's the syntax

Reply:
I am not sure that we can or should link from the markdown to the doxygen class documentation. I think for this document, it is sufficient to mention the class without a link.